torch_audiomentations/augmentations/background_noise.py

```diff
@@ -4,113 +4,143 @@
 import numpy as np
 import soundfile
 import torch
-import typing
+from typing import Union, List
+from pathlib import Path
 
 from ..core.transforms_interface import BaseWaveformTransform, EmptyPathException
 from ..utils.dsp import calculate_rms, calculate_desired_noise_rms
-from ..utils.file import find_audio_files, load_audio
+from ..utils.file import find_audio_files
+from ..utils.io import Audio
 
 
 class ApplyBackgroundNoise(BaseWaveformTransform):
     """
     Add background noise to the input audio.
+
     """
 
-    supports_multichannel = False  # TODO: Implement multichannel support
+    supports_multichannel = True  # TODO: Implement multichannel support
+    requires_sample_rate = True
 
     def __init__(
         self,
-        bg_path,
+        background_paths: Union[List[Path], Path],
         min_snr_in_db: float = 3.0,
         max_snr_in_db: float = 30.0,
-        device=torch.device("cpu"),
         mode: str = "per_example",
         p: float = 0.5,
-        p_mode: typing.Optional[str] = None,
-        sample_rate: typing.Optional[int] = None,
+        p_mode: str = None,
+        sample_rate: int = None,
```

Comment:
Why remove typing.Optional?

Reply:
Because, in my understanding, this is more readable andequivalent: `mypy` will automatically infer `Optional` when default value is `None`. But I don't mind reverting this change if you ask me to.

Reply:
What does this thumb up mean? Should I keep `Optional`?